pkg/server/config.go

```diff
@@ -39,6 +41,13 @@ func (cfg *HandlerConfig) Validate() error {
 		return fmt.Errorf("PMAP_SUCCESS_TOPIC_ID is empty and requires a value")
 	}
 
+	if cfg.ResourceScope != "" &&
+		!strings.HasPrefix(cfg.ResourceScope, "projects/") &&
```

Comment:
We should make allowed prefixes a global var.

Reply:
I thought about making it a global var in `const()`, but they are only used here, so I though putting it here will make the prefix more obvious.